src/NuGet.Core/NuGet.Configuration/NuGet.Configuration.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>NuGet's configuration settings implementation.</Description>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
-    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
```

Comment:
Doesn't IsBuildOnlyXPLATProjects default to DotNetBuildFromSource anyways?

Reply:
I asked for this change. `IsBuildOnlyXPLATProjects` was nothing more than an alias for `DotNetBuildFromSource`, so I asked for it to be removed.

In particular, I know that in our build scripts we have some logic when building on Linux/Mac to do some things differently than when building on Windows. From memory there's a property named something like IsXPLAT?  I personally found it confusing what is source build specific, and what is our own Windows/Linux. By keeping the source build logic using a condition named BuildFromSource, then it's much clearer what the intent between the different conditions and variables are.

Reply:
The original idea was that you don't have to care what's source build specific because a large portion of it overlapped with other needs.

I don't care too much.

Reply:
Yes, as @zivkan mentioned about though this is kind of a holdover from when we had different properties for `DotNetBuildFromSource` - this isn't really one of our recommendations anymore.  I can go either way though.